contracts/gnosis/EIP4337Fallback.sol

```diff
@@ -30,4 +34,20 @@ contract EIP4337Fallback is DefaultCallbackHandler, IAccount {
         return abi.decode(ret, (uint256));
     }
 
+    function isValidSignature(
+        bytes32 _hash,
+        bytes memory _signature
+    ) external override view returns (bytes4) {
+        bytes32 hash = _hash.toEthSignedMessageHash();
+        address recovered = hash.recover(_signature);
+
+        GnosisSafe safe = GnosisSafe(payable(address(msg.sender)));
+
+        // Validate signatures
+        if (safe.isOwner(recovered)) {
+            return 0x1626ba7e;
```

Comment:
can you please extract this constant (also in the test code) ?
I prefer not to see "anonymous" magic values in code...
(0xffff is good enough, since its value is as good as a constant name...)

Reply:
Also, it would be nice to add such implementation to BaseAccount, for the benefit of other wallet types (though it should be a separate PR)